bridge/opentracing/bridge.go

```diff
@@ -620,6 +620,25 @@ func (s fakeSpan) SpanContext() trace.SpanContext {
 	return s.sc
 }
 
+// adapt OpenTracing TextMapWriter to OpenTelemetry propagation.TextMapCarrier
+type textMapWriterAdapter struct {
+	t ot.TextMapWriter
+}
+
+func (a textMapWriterAdapter) Get(key string) string {
+	// not implemented
+	return ""
+}
+
+func (a textMapWriterAdapter) Set(key string, value string) {
+	a.t.Set(key, value)
+}
+
+func (a textMapWriterAdapter) Keys() []string {
+	// not implemented
+	return nil
+}
+
 // Inject is a part of the implementation of the OpenTracing Tracer
 // interface.
 //
```

Comment:
Docs need to be updated that they support more than HTTP.

Reply:
Is this a reference to the line below?  The `format` parameter is still checked to be `ot.HTTPHeaders`, here:
https://github.com/open-telemetry/opentelemetry-go/pull/2141/files#diff-0f0430a4013bb87f6c4d12fa2f8a997e871e952c5a3f7f279aff88b54eb1cf32R674-R676